lib/coverband/utils/results.rb

```diff
@@ -20,6 +20,22 @@ def file_with_type(source_file, results_type)
         get_results(results_type).source_files.find { |file| file.filename == source_file.filename }
       end
 
+      def runtime_relevant_coverage(source_file)
+        eager_loading_coverage = get_results(Coverband::EAGER_TYPE)
+        runtime_coverage = get_results(Coverband::RUNTIME_TYPE)
+        return unless eager_loading_coverage && runtime_coverage
+
+        eager_file = eager_loading_coverage.source_files.find { |file| file.filename == source_file.filename }
+        runtime_file = runtime_coverage.source_files.find { |file| file.filename == source_file.filename }
+
+        return 0.0 unless runtime_file
+
+        return 100.0 unless eager_file
+
+        runtime_relavant_lines = eager_file.relevant_lines - eager_file.covered_lines_count
+        runtime_file.runtime_relavant_calculations(runtime_relavant_lines, &:formatted_covered_percent)
```

Comment:
I am having a hard time understanding this part. Could be because I am not as familiar with this area of the codebase. Seems like we are passing in a proc as the second argument but it's unclear what is passed into the proc unless we dive into the runtime_relavant_calculations implementation. Feels a little advanced to me but maybe it's needed?

Reply:
ha check git history I wrote this differently and rubocop yelled at me, so I actually think the original implementation is more readable if you agree I will go back to that and disable the robucop warning.

Reply:
Agree that the original implementation was more readable.

Reply:
ok cool reverted back to that implementation and updated rubocop to not complain about that.